storybook/src/elements/nuxeo-actions-menu/nuxeo-actions-menu.stories.js

```diff
@@ -0,0 +1,23 @@
+import { storiesOf } from '@storybook/polymer';
+import { number } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-actions-menu';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
+
+storiesOf('UI/nuxeo-actions-menu', module).add(
+  'Default',
+  () => html`
+    <style>
+      nuxeo-actions-menu {
+        max-width: 300px;
+      }
+    </style>
+    <nuxeo-actions-menu>
+      ${Array.from({ length: number('# actions', 10) }).map(
+        () => html`
```

Comment:
If I understand correctly the idea here is to create an action menu with `10` points, my point is what if we replace directly with:
`Array.from({length: 10})` which will save the import of `import { number } from '@storybook/addon-knobs';`

Reply:
I think the idea was the knob was there for the user to control how many menu items he wanted to see, not sure if we want to keep that or not.

Reply:
Indeed, Andrea!
We are here playing with Storybook knobs.

Reply:
yes I see, my question how many of `nuxeo-link-button` we will have in `nuxeo-actions-menu`

Reply:
By default: 10
And then according to the user's input, it will render X amount of `nuxeo-link-button`.
You can take a ride here: https://nuxeo.github.io/nuxeo-elements/?path=/story/ui-widgets--nuxeo-actions-menu

Reply:
Thanks Nuno :)